packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -804,23 +805,53 @@ class RawGestureDetector extends StatefulWidget {
   /// duplication of information.
   final bool excludeFromSemantics;
 
+  /// Describes the semantics notations that should be added to the underlying
+  /// render object [RenderSemanticsGestureHandler].
+  ///
+  /// It has no effect if [excludeFromSemantics] is true.
+  ///
+  /// When [semantics] is null, [RawGestureDetector] will fall back to
+  /// default delegate that behaves as follows:
+  ///
+  ///  * During a semantic tap, it looks for [TapGestureRecognizer] and calls
+  ///    `onTapDown`, `onTapUp`, and `onTap`.
+  ///  * During a semantic long press, it looks for [LongPressGestureRecognizer]
+  ///    and calls `onLongPressStart`, `onLongPress`, `onLongPressEnd` and
+  ///    `onLongPressUp`.
+  ///  * During a semantic horizontal drag, it looks for [HorizontalDragGestureRecognizer]
+  ///    and calls `onDown`, `onStart`, `onUpdate` and `onEnd`. Then it looks
+  ///    for [PanGestureRecognizer] and calls the same methods.
+  ///  * During a semantic vertical drag, it looks for [VerticalDragGestureRecognizer]
+  ///    and calls `onDown`, `onStart`, `onUpdate` and `onEnd`. Then it looks
+  ///    for [PanGestureRecognizer] and calls the same methods.
+  final SemanticsGestureDelegate semantics;
+
   @override
   RawGestureDetectorState createState() => RawGestureDetectorState();
 }
 
 /// State for a [RawGestureDetector].
 class RawGestureDetectorState extends State<RawGestureDetector> {
   Map<Type, GestureRecognizer> _recognizers = const <Type, GestureRecognizer>{};
+  SemanticsGestureDelegate _semantics;
 
   @override
   void initState() {
     super.initState();
+    _semantics = widget.semantics ??
```

Comment:
nit: just leave this in one line for easier readability?

Reply:
Fixed